tools/topology/platform/intel/jsl-rt1015.m4

```diff
@@ -12,7 +12,7 @@ define(`SPK_DATA_FORMAT', `s24le')
 
 define(`SET_SSP_CONFIG',
 				`SSP_CONFIG(I2S, SSP_CLOCK(mclk, 24000000, codec_mclk_in),
-						SSP_CLOCK(bclk, 2400000, codec_slave),
+						SSP_CLOCK(bclk, 4800000, codec_slave),
 						SSP_CLOCK(fsync, 48000, codec_slave),
-						SSP_TDM(2, 25, 3, 3),
+						SSP_TDM(2, 32, 3, 3),
```

Comment:
this seems wrong with a mismatch between slot size and bclk.

32 * 2 * 48000 = 3072000. You selected 4800000.

??

Reply:
also please fix your commit title, should be topology: jsl-rt1015: PLL input at 50fs is no longer supported

Reply:
Changes are done